### DISCUSSION TITLE
api余额问题

### DISCUSSION THREAD

**Discussion by Li1177** [ChatGPTNextWeb/NextChat#2593] [Q&A]
ChatGPT-Next-Web 是我用过的最好用的项目，没有之一。 最近突然报错，似乎是说我的余额没有了，但是我非常确定我登录openai
查看api余额的还有1.2美元的额度。 （我的一个免费5美元的账号，到期日是10月1日）
是前天发生的，我的上一个问题还在正常回答，结果第二个问题就报错：
{
  "error": {
    "message": "You exceeded your current quota, please check your plan and billing details.",
    "type": "insufficient_quota",
    "param": null,
    "code": "insufficient_quota"
  }
}

这有可能是什么情况啊？

**Comment by 50476726** [comment]
我也遇到了

**Reply by Li1177** [reply]
我还有一个付费API账户，上个月还支付了几美元的gpt3.5费用，这个月还不给我的GPT4，尴尬， 不舍得用。

**Reply by Li1177** [reply]
我知道问题是什么了，看返回的代码怎么看都像是openai的问题， 但是实际上我们自己手动更新一下ChatGPT-Next-Web就可以了

**Reply by aowu00** [reply]
各位大佬 上面显示余额不足的问题 怎么解决 ，在chat官网还能正常使用啊 为什么使用api到这就不能使用了

**Reply by mxiaonian** [reply]
一样，在官网能使用，使用apikey到这个客户端，就显示无余额

**Reply by KarnieNeko** [reply]
同问，还是说免费帐户的api没法用？

**Reply by Jason7187** [reply]
怎么查看余额

**Comment by GOWxx** [comment]
设置授权码，不然会被白嫖。

**Comment by mxiaonian** [comment]
各位大佬解决没有呢？我使用的是本地客户段，配置API KEY后，也是提示余额不足，3.5的模型都不能用

**Comment by wanghongxingdg** [comment]
付费的网址是多少， 突然找不到啦
